bucket/everything-cli.json

```
@@ -1,12 +1,18 @@
 {
-    "homepage": "https://www.voidtools.com/support/everything/command_line_interface/",
+    "version": "1.1.0.23",
     "description": "Command line interface to Everyting.",
+    "homepage": "https://www.voidtools.com/support/everything/command_line_interface/",
     "license": "MIT",
-    "version": "1.1.0.23",
     "depends": "extras/everything",
     "url": "https://www.voidtools.com/ES-1.1.0.23.zip",
     "hash": "e8ac817ae276ba1d12044a65444c109f284015ba985d29201a9fd4f38b5e6187",
+    "pre_install": [
+        "$manifest.persist | ForEach-Object {",
+        "    if (-not (Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" -Force -ItemType file | Out-Null }",
```

```suggestion
        "    if (-not (Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" -ItemType file | Out-Null }",
```
Force shouldn't be required